ios/RCTWebRTC/WebRTCModule+RTCPeerConnection.m

```diff
@@ -309,6 +309,31 @@ @implementation WebRTCModule (RTCPeerConnection)
                 }];
 }
 
+RCT_EXPORT_METHOD(peerConnectionSendDTMF:(nonnull NSString *)tone duration:(NSTimeInterval)duration interToneGap:(NSTimeInterval)interToneGap objectID:(nonnull NSNumber *)objectID)
+{
+  RTCRtpSender* audioSender = nil ;
+  RTCPeerConnection *peerConnection = self.peerConnections[objectID];
+  if (!peerConnection) {
+    NSLog(@"peerConnectionSendDTMF() peerConnection is nil");
+    return;
+  }
+  for (RTCRtpSender *sender in peerConnection.senders){
+    if([[[sender track] kind] isEqualToString:@"audio"]) {
+       audioSender = sender;
+     } 
+  }
+  if(audioSender){
+    NSOperationQueue *queue = [[NSOperationQueue alloc] init];
```

Comment:
Why are you using this queue here?

Reply:
I don't know, I take code from here https://stackoverflow.com/a/60148372 , I removed it now

Reply:
I think we should use queue here, [see here](https://chromium.googlesource.com/external/webrtc/+/HEAD/sdk/android/api/org/webrtc/DtmfSender.java#30)
It says 
```
 Queues a task that sends the provided DTMF tones.
 If insertDtmf is called on the same object while an existing task for this
 object to generate DTMF is still running, the previous task is canceled.
```
So what should we do?